content/navigate-history.en.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Browsing through terms"
+weight: 10
+---
+
+# Browsing through terms
+
+Every collection offers a public database of versions they recorded. 
+
+For this guide, we will use the example of the demo collection. This collection publishes its history on the [`OpenTermsArchive/demo-versions`](https://github.com/OpenTermsArchive/demo-versions) repository.
+
+- From the [repository page](https://github.com/OpenTermsArchive/demo-versions), open the folder of the service of your choice by clicking on it. For example, [GitHub](https://github.com/OpenTermsArchive/demo-versions/tree/main/GitHub):
```

Comment:
> I would have preferred that we use Open Terms Archive as an example. It's okay to keep GitHub now that all the images have been done but, for future examples, the most stable example we can give is Open Terms Archive.

Reply:
I did it with Open Terms Terms at first but I encountered problems because our files did not changed and we did not have history. So I re do everything with GitHub…